packages/react-components/react-button/src/components/Button/useButtonStyles.styles.ts

```diff
@@ -268,12 +268,12 @@ const useRootStyles = makeStyles({
 
     '@media (forced-colors: active)': {
       ':hover': {
-        backgroundColor: tokens.colorTransparentBackgroundHover,
+        backgroundColor: 'transparent',
```

Comment:
I just checked and you can achieve the same thing using `backgroundColor: tokens.colorTransparentBackground` (not hover) instead of using `'transparent'` directly:

```suggestion
        backgroundColor: tokens.colorTransparentBackground,
```